packages/ses/src/module-load.js

```diff
@@ -38,21 +41,37 @@ const resolveAll = (imports, resolveHook, fullReferrerSpecifier) => {
 // This graph is then ready to be synchronously linked and executed.
 export const load = async (
   compartmentPrivateFields,
+  moduleAliases,
   compartment,
   moduleSpecifier,
 ) => {
   const {
     resolveHook,
     importHook,
-    aliases,
+    moduleMap,
     moduleRecords,
   } = compartmentPrivateFields.get(compartment);
 
-  // Follow aliases.
-  if (aliases.has(moduleSpecifier)) {
-    const alias = aliases.get(moduleSpecifier);
+  // Follow moduleMap.
+  const aliasNamespace = moduleMap[moduleSpecifier];
```

Comment:
The subsequent checks are redundant, but won’t be after this `moduleMap` lookup 👆 is followed by a dynamic `moduleMapHook` call.